src/client/components/Account/AccountInfo.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+
+// components
+import { Button } from '../generic/Button';
+
+// util
+import { getToken } from '../../utils/auth';
+
+interface AuthProvidorId {
+  provusername: string | undefined;
```

Comment:
Why not use the syntax `provusername?: string`? Not a significant difference to me, but I'm just curious

Reply:
cuz it's not an option.  It "should be" there is if there is an Auth provider.  Putting undefined just because IF one of the data (like just provider username) is missing, and there is no "undefined", it might complain and not even load the page.  It "shouldn't" tho.  I took out the "| undefined" to reduce confusion.